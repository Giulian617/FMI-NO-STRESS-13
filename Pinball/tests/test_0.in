5 0
3 4 7
2 4 7
1 2 6
5 4 6
