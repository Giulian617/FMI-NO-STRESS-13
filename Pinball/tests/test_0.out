0
76923078
923076930
